pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 119
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "0.1.15"
+version = "0.1.16"
```

Comment:
Looks like you're a couple of versions behind btw. Might need to rebase `main`.

https://github.com/uktrade/copilot-tools/blob/main/pyproject.toml#L6

Reply:
This has been bumped to 0.1.19